docs/rest.md

```diff
@@ -514,3 +514,46 @@ Comma separated list of notification types, either `logging` or `operations`.
 #### resource
 
 Generally the UUID of a background operation you are interested in.
+
+## /v2/buy
+
+### POST
+
+* Description: Buy the specified snap
+* Access: authenticated
+* Operation: sync
+* Return: Dict with buy state.
+
+#### Sample input using default payment method:
+
+```javascript
+{
+    "snap-id": "2kkitQurgOkL3foImG4wDwn9CIANuHlt",
+    "snap-name": "moon-buggy",
+    "channel": "moon-buggy",
+    "expected-price": "2.99",
```

Comment:
Can we drop the "expected-" prefix from here?  price+currency feels unambiguous on a buy payload.